data/training-modules.yml

```diff
@@ -15,3 +15,9 @@ personal-social-and-emotional-development:
   thumbnail: introduction_to_supporting_physical_development
   description: about personal-social-and-emotional-development
   depends_on: brain-development-and-how-children-learn
+
```

Comment:
Is this just an example?  The summative questions go at the end of each module.

Reply:
I'd add that any work in progress need be done on the `demo-modules` first. You can toggle what gets published in your review app by commenting the `app_config.yml` in terraform